script.js

```diff
@@ -1,4 +1,76 @@
+// Basic Scissors Paper Stone Project
+
+var scissors = `scissors`;
+var paper = `paper`;
+var stone = `stone`;
+var computerChoice = function (generateRandomNumber) {
+  return generateRandomNumber;
+};
+var myOutputValue = `Please select either scissors, paper or stone to start the game.`;
+
+//Computer Choice
+var generateRandomNumber = function () {
+  var randomDecimal = Math.random() * 3;
+  var randomNumber = Math.floor(randomDecimal);
+  if (randomNumber == 0) {
+    return scissors;
+  }
+  if (randomNumber == 1) {
+    return paper;
+  }
+  if (randomNumber == 2) {
+    return stone;
+  }
+};
```

Comment:
Look at https://replit.com/@kenrickles/Difference-between-MultipleIfs-and-If-else-Ifs#index.js for more information on using multiple ifs.